Makefile

```diff
@@ -1,4 +1,5 @@
 ###############################################################################
+<<<<<<< HEAD
```

Comment:
Conflict here

Reply:
I thought I got those. Oops. Cleaning and repushing now.

Reply:
I *did* clean it up. And git should not have accepted it without it. Oh well, will fix.

Reply:
Updated. Please review @fasaxc